Oryx.sln

```diff
@@ -96,7 +96,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Common", "src\Oryx.Common\C
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BuildScriptGenerator.Common.Tests", "tests\BuildScriptGenerator.Common.Tests\BuildScriptGenerator.Common.Tests.csproj", "{3078BCDB-FC35-4826-82A1-9CE094446DE8}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoUpdater", "build\tools\AutoUpdater\AutoUpdater.csproj", "{6C4DBA89-3D17-451C-9D89-71A70410D835}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AutoUpdater", "build\tools\AutoUpdater\AutoUpdater.csproj", "{6C4DBA89-3D17-451C-9D89-71A70410D835}"
```

Comment:
Is AutoUpdater.csproj a project you're currently working on? If so, do we consider make it as a seperate PR or will add into this one afterwards?

Reply:
No, it has been removed now.